index.html

```diff
@@ -1,140 +1,170 @@
 <!DOCTYPE html>
 <html lang="en">
-	<head>
-		<meta charset="utf-8">
-		<meta http-equiv="X-UA-Compatible" content="IE=edge">
-		<title>Git for Windows</title>
-		<meta name="description" content="We bring the awesome Git VCS to Windows">
-		<meta name="viewport" content="width=device-width, initial-scale=1">
-		<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700">
-		<link rel="stylesheet" href="css/pack.css">
-		<link rel="shortcut icon" href="favicon.ico">
-		<!--[if lt IE 9]>
+
+<head>
```

Comment:
Could we keep the original indentation? I find it really hard to pick out the actual changes between the indentation changes...

Reply:
Hmmm, I was using `vscode` for this, I think the prettier or prettify formatted these. I'll see what I can do to make it easy for review.

Reply:
Maybe start with a commit that adjusts _just_ the indentation, but does not change anything else?